### PR TITLE
[Edge WS ] Objective

### PR DESCRIPTION
**Pull request by ASawwaf**
@bfcohen @markshostak @edhorwath 
can you have look at this

### CODE REVIEW

**Comment by ASawwaf** [comment]
@bfcohen @markshostak @edhorwath

any comment to merge it

File: doc/tech/usecases.md [hunk 1]
```diff
@@ -11,3 +11,16 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+
+2-Objective :
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:16]: I think this should be 5.1.2

File: doc/tech/usecases.md [hunk 2]
```diff
@@ -11,3 +11,16 @@
...
+
+
+Based on CNTT’s Project goals & Purpose was formulated is to develop a robust infrastructure model and define a limited discrete set of architectures built on that model that can be tested and validated for use across the entire member community. 
+
+CNTT Scope is extending beyond the Cloud infrastructure that should build into the Regional or National Data Center  but it extended to Edge , which CNTT consider it as a disruptive  approach and use case that CNTT will add value especially there is a lot of scattered project/Initiative either under Standard Development bodies or  Open source communities but there is no
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:21]: The last sentence is not finished.

File: doc/tech/usecases.md [hunk 3]
```diff
@@ -11,3 +11,16 @@
...
+
+
+2-Objective :
+
+
+Based on CNTT’s Project goals & Purpose was formulated is to develop a robust infrastructure model and define a limited discrete set of architectures built on that model that can be tested and validated for use across the entire member community. 
+
+CNTT Scope is extending beyond the Cloud infrastructure that should build into the Regional or National Data Center  but it extended to Edge , which CNTT consider it as a disruptive  approach and use case that CNTT will add value especially there is a lot of scattered project/Initiative either under Standard Development bodies or  Open source communities but there is no 
+
+So Object of this Edge is :
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:23]: Edge related activity of Edge work stream.

File: doc/tech/usecases.md [hunk 4]
```diff
@@ -11,3 +11,16 @@
...
+-	To Harmonize the work under Standard Development bodies or  Open source communities 
+-	To build Common Cloud infrastructure based on CNTT principles that can be consumed by any operator
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:25]: Modify the existing RM and RA-s so that they are aligned with the edge requirements

Comment (ASawwaf) [reply doc/tech/usecases.md:25]: done

File: doc/tech/usecases.md (hunk 3 above)
Comment (ASawwaf) [reply doc/tech/usecases.md]: done

File: doc/tech/usecases.md (hunk 2 above)
Comment (ASawwaf) [reply doc/tech/usecases.md]: done

File: doc/tech/usecases.md [hunk 5]
```diff
@@ -11,3 +11,17 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+
+2-Objective :
+
+
+Based on CNTT’s Project goals & Purpose was formulated is to develop a robust infrastructure model and define a limited discrete set of architectures built on that model that can be tested and validated for use across the entire member community.
```
Comment (pgoyal01) [inline doc/tech/usecases.md:19]: Suggest change:
"Based on CNTT’s Project goals & Purpose was formulated is to develop a robust infrastructure model and define a limited discrete set of architectures ..." --> "CNTT’s goal and purpose is to develop a robust cloud infrastructure model and define a limited set of discrete architectures ..."

File: doc/tech/usecases.md [hunk 6]
```diff
@@ -11,3 +11,17 @@
...
 <a name="5.1"></a>
 ## 5.1 Edge
+
+
+2-Objective :
+
+
+Based on CNTT’s Project goals & Purpose was formulated is to develop a robust infrastructure model and define a limited discrete set of architectures built on that model that can be tested and validated for use across the entire member community. 
+
+CNTT Scope is extending beyond the Cloud infrastructure that should build into the Regional or National Data Center  but it extended to the Edge , which CNTT consider it as a disruptive  approach and use case that CNTT will add value especially there is a lot of scattered project/Initiative either under Standard Development bodies or  Open source communities
```
Comment (pgoyal01) [inline doc/tech/usecases.md:21]: Suggest change:
"CNTT Scope is extending beyond the Cloud infrastructure that should build into the Regional or National Data Center  but it extended to the Edge , which CNTT consider it as a disruptive  approach and use case that CNTT will add value especially there is a lot of scattered project/Initiative either under Standard Development bodies or  Open source communities"

"Extend CNTT's Scope beyond the Regional and National Data Center cloud infrastructures to the Edge. The Edge is a disruptive use case where CNTT can add value especially as there are a number of scattered initiatives under various Standard Development Organisations (SDO) and open source communities."

File: doc/tech/usecases.md [hunk 7]
```diff
@@ -11,3 +11,17 @@
...
+
+Edge Stream related activities :
+-	To Harmonize the work under Standard Development bodies or  Open source communities
```
Comment (pgoyal01) [inline doc/tech/usecases.md:24]: Are we really going to do this or maybe I am reading too much into "harmonize"

File: doc/tech/usecases.md [hunk 8]
```diff
@@ -11,3 +11,17 @@
...
+
+
+Based on CNTT’s Project goals & Purpose was formulated is to develop a robust infrastructure model and define a limited discrete set of architectures built on that model that can be tested and validated for use across the entire member community. 
+
+CNTT Scope is extending beyond the Cloud infrastructure that should build into the Regional or National Data Center  but it extended to the Edge , which CNTT consider it as a disruptive  approach and use case that CNTT will add value especially there is a lot of scattered project/Initiative either under Standard Development bodies or  Open source communities
+
+Edge Stream related activities :
+-	To Harmonize the work under Standard Development bodies or  Open source communities 
+-	To build Common Cloud infrastructure based on CNTT principles that can be consumed by any operator
+-	To build Cloud infrastructure that can scale and cover the Edge Telco use cases that can help CSP to monetize the NFV/SDN journey
```
Comment (pgoyal01) [inline doc/tech/usecases.md:26]: suggest "scale over hundreds of thousands of nodes"
CSP acronym first use -- we have also been using "operator"

**Comment by rabi-abdel** [comment]
content moved to https://github.com/cntt-n/CNTT/pull/1449 to avoid confusion.

**Pull request closed without merging**